src/GraphQL/Utilities/ISchemaNodeVisitor.cs

```diff
@@ -5,9 +5,18 @@ namespace GraphQL.Utilities
 {
     public interface ISchemaNodeVisitor
     {
-        void VisitObjectGraphType(ObjectGraphType type);
+        void VisitSchema(Schema schema);
+        void VisitScalar(ScalarGraphType scalar);
+        void VisitObject(ObjectGraphType type);
+        void VisitObject(IObjectGraphType type);
```

Comment:
ObjectGraphType and IObjectGraphTpe are the same thing. You do not need both.

Reply:
We had some doubts. Which one is better to leave and which to remove?

Reply:
`VisitObject(ObjectGraphType type)` is not enough because both `ObjectGraphType` and `ObjectGraphType<TSource>` can have directives and `ObjectGraphType : ObjectGraphType<object>`.
So remove `void VisitObject(ObjectGraphType type)`.

Reply:
I am for removing ObjectGraphType at all #1331